pkg/roachprod/roachprod.go

```diff
@@ -1034,13 +1034,19 @@ func Reformat(ctx context.Context, l *logger.Logger, clusterName string, fs stri
 
 	var fsCmd string
 	switch fs {
-	case vm.Zfs:
-		if err := install.Install(ctx, l, c, []string{vm.Zfs}); err != nil {
+	case string(vm.Zfs):
```

Comment:
Similar nit: would it be cleaner to just cast fs to a filesystem? `switch vm.Filesystem(fs) {`